app/controllers/stats_controller.rb

```diff
@@ -1,7 +1,7 @@
 class StatsController < ApplicationController
   def subscriptions
     @tags = {}
-    TagSelection.where(following: true).each do |tag|
+    TagSelection.joins(:node_tags).where(following: true).each do |tag|
```

Comment:
Thanks for working on this so quick @dongskyler :tada:  ...we want this to become `TagSelection.where(following: true)        .joins(:node_tags)` as explained in this comment https://github.com/publiclab/plots2/issues/7831#issuecomment-620770359

Reply:
I’m happy to work on it shortly. 

But to be clear, this calls for another branch and another PR, and it will void my previous PR. This was not what I modified in my previous branch.

Reply:
Not really...we are moving the 'join' after the 'where'..
in the previous PR we had `TagSelection.joins(:node_tags).where(following: true)` and before that we did not have the joins to node_tags

Reply:
I misunderstood what you instructed me to do.

I have updated the branch and both tests have passed.

Please review.